app/views/content/funding-and-support/if-youre-disabled.md

```diff
@@ -36,17 +36,19 @@ If you’re disabled, have a mental health condition or educational needs, you c
 
 $international-content$
 
-## Disabled Students’ Allowance and other support
+## Financial support
+
+### Disabled Students' Allowance (DSA)
 
 You can [apply for Disabled Students’ Allowance (DSA)](https://www.gov.uk/disabled-students-allowance-dsa/how-to-claim) to cover the study-related costs you have because of a mental health problem, long term illness or any other disability.
 
 You’ll get confirmation of whether your application has been successful within 6 weeks.
 
 It can take up to 14 weeks to get your DSA support, so you should take this into consideration if you need the support in place for the start of your course.
 
-[If you do paid training you may also be eligible for support](https://www.gov.uk/access-to-work).
+### Other support for funding your training
```

Comment:
This feels slightly clunky - could we say something like `Other funding and support` (since Access to Work covers more than just financial support)?

Reply:
```suggestion
### Other funding and support
```